lib/autocomplete-manager.js

```diff
@@ -37,14 +37,7 @@ class AutocompleteManager {
     this.shouldDisplaySuggestions = false
     this.prefixRegex = null
     this.wordPrefixRegex = null
-    this.updateCurrentEditor = this.updateCurrentEditor.bind(this)
```

Comment:
This change may break code! These functions always keep there `this` even if they are passed to somewhere else. You would first need to check if this behavior is used somewhere.